hack/test-go.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+source "$(dirname "${BASH_SOURCE}")/init.sh"
```

Comment:
/me just noticed for the first time that you can nest `""` inside `$()` inside `""`...

Reply:
Yeah I noticed that this weekend, I was also surprised.